# Input
# 0 - end jog
# 1 - start jog
bool IO
---
bool success